Develop an SQL schema for SOPs

```diff
@@ -0,0 +1,4351 @@
+
+
+--
+-- TOC entry 2832 (class 2606 OID 18255)
+-- Name: Small Molecule Evidence (SME) Section Small Molecule Evidence (SME) Section_SME_ID_fkey; Type: FK CONSTRAINT; Schema: MzTab-M; Owner: postgres
+--
+
+ALTER TABLE ONLY "MzTab-M"."Small Molecule Evidence (SME) Section"
+    ADD CONSTRAINT "Small Molecule Evidence (SME) Section_SME_ID_fkey" FOREIGN KEY ("SME_ID") REFERENCES "MzTab-M"."Small Molecule Feature (SMF) Section"("SME_ID_REFS") NOT VALID;
+
+
+--
+-- TOC entry 2833 (class 2606 OID 18248)
+-- Name: Small Molecule Feature (SMF) Section Small Molecule Feature (SMF) Section_SMF_ID_fkey; Type: FK CONSTRAINT; Schema: MzTab-M; Owner: postgres
+--
+
+ALTER TABLE ONLY "MzTab-M"."Small Molecule Feature (SMF) Section"
+    ADD CONSTRAINT "Small Molecule Feature (SMF) Section_SMF_ID_fkey" FOREIGN KEY ("SMF_ID") REFERENCES "MzTab-M"."Small Molecule Section"("SMF_ID_REFS") NOT VALID;
+
+
+--
+-- TOC entry 2828 (class 2606 OID 18261)
+-- Name: Small Molecule Section Small Molecule Section_SMF_ID_REFS_fkey; Type: FK CONSTRAINT; Schema: MzTab-M; Owner: postgres
+--
+
+ALTER TABLE ONLY "MzTab-M"."Small Molecule Section"
+    ADD CONSTRAINT "Small Molecule Section_SMF_ID_REFS_fkey" FOREIGN KEY ("SMF_ID_REFS") REFERENCES "MzTab-M"."Small Molecule Feature (SMF) Section"("SMF_ID") NOT VALID;
+
+
+-- Completed on 2020-10-30 17:54:26
+
+--
+-- PostgreSQL database dump complete
+--
+
+--
+-- PostgreSQL database dump
+--
+
+-- Dumped from database version 12.3
+-- Dumped by pg_dump version 12.3
+
+-- Started on 2020-10-30 17:12:34
+
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET idle_in_transaction_session_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SELECT pg_catalog.set_config('search_path', '', false);
+SET check_function_bodies = false;
+SET xmloption = content;
+SET client_min_messages = warning;
+SET row_security = off;
+
+DROP DATABASE "MFA analysis";
+--
+-- TOC entry 3134 (class 1262 OID 17718)
+-- Name: MFA analysis; Type: DATABASE; Schema: -; Owner: postgres
+--
+
+CREATE DATABASE "MFA analysis" WITH TEMPLATE = template0 ENCODING = 'UTF8' LC_COLLATE = 'German_Germany.1252' LC_CTYPE = 'German_Germany.1252';
+
+
+ALTER DATABASE "MFA analysis" OWNER TO postgres;
+
+\connect -reuse-previous=on "dbname='MFA analysis'"
+
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET idle_in_transaction_session_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SELECT pg_catalog.set_config('search_path', '', false);
+SET check_function_bodies = false;
+SET xmloption = content;
+SET client_min_messages = warning;
+SET row_security = off;
+
+SET default_tablespace = '';
+
+SET default_table_access_method = heap;
+
+--
+-- TOC entry 209 (class 1259 OID 17752)
+-- Name: MDVFragments_from_excel; Type: TABLE; Schema: public; Owner: postgres
+--
+
+CREATE TABLE public."MDVFragments_from_excel" (
+    met_id text NOT NULL,
+    formula text NOT NULL,
+    elements text[] NOT NULL,
```

Comment:
Let's discuss whether it makes sense to change the table schema so that we no longer have a list here.